cluster/examples/kubernetes/ceph/cluster-test.yaml

```diff
@@ -14,7 +14,7 @@ metadata:
   namespace: rook-ceph
 spec:
   cephVersion:
-    image: ceph/ceph:v14.2.1-20190430
+    image: ceph/ceph:latest-nautlius-devel
```

Comment:
typo: nautlius

Reply:
the repo name and the tag name are incorrect.

It's ceph/daemon-base:latest-nautilus-devel